src/Components/User/User.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../styles/variables";
+
+.user-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  background-color: $main-white;
+  width: 25em;
+  margin: 2em;
+  padding: 1em;
+}
+
+.profile-picture {
+  border: 5px solid $main-orange;
+  border-radius: 50%;
+  height: 10em;
+  width: 10em;
+}
+
+.section-titles {
+  width: 100%;
+  margin: 2em;
+}
+
+.hours-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: $main-white;
+  background-color: $main-blue;
+  height: 2em;
+  width: 50%;
+  margin: 0.5em auto;
```

Comment:
We need to create more variables or mixins to avoid repetitiveness like margin: 0.5em